convert2rhel_insights_tasks/main.py

```diff
@@ -701,8 +707,15 @@ def run_convert2rhel(env):
     if not bool(els_disabled):
```

Comment:
```suggestion
    if not els_disabled:
```
The json.loads already returns a boolean.

Reply:
Ah, that's true. I can remove this in the next patch. It won't harm the script